pytest_splinter/plugin.py

```diff
@@ -287,16 +287,18 @@ def splinter_browser_class(request):
     return Browser
 
 
-def get_args(driver=None,
-             download_dir=None,
-             download_ftypes=None,
-             firefox_pref=None,
-             firefox_prof_dir=None,
-             remote_url=None,
-             executable=None,
-             headless=False,
-             driver_kwargs=None):
+def get_args(request):
```

Comment:
I'd keep get_args independent from knowing how to access fixtures from a request. Please keep the old signature